analytics_dashboard/courses/templates/courses/home.html

```diff
@@ -32,26 +32,20 @@
   <div class="row course-home-table-outer">
     {% for column in table_items %}
       <div class="col-sm-4">
-        <header class="heading-outer">
-          <div class="heading">{{ column.heading }}</div>
-        </header>
-
         <div class="course-home-table">
           <div class="item name">
-            <i class="ico fa {{ column.icon }}" aria-hidden="true"></i> {{ column.name }}
+            <i class="ico fa fa-fw {{ column.icon }}" aria-hidden="true"></i> {{ column.name }}
+            <div class="sub-heading"><small>{{ column.heading }}</small></div>
           </div>
 
           {% for item in column.items %}
             <div class="item">
               <div class="title"><a href="{% url item.view course_id=course_id %}">{{ item.title }}</a></div>
               <div class="breadcrumbs">
-                <i class="ico fa {{ column.icon }}" aria-hidden="true"></i>
-                <i class="ico fa fa-caret-right" aria-hidden="true"></i>
```

Comment:
Can we use our theme css to change the fa-caret-right to a slash symbol instead of changing the template?